tests/test_routes.py

```diff
@@ -187,6 +187,22 @@ def test_delete_nonexist_product(self):
         self.assertEqual(response.status_code, status.HTTP_204_NO_CONTENT)
         logging.debug("not found")
 
+    def test_update_pet(self):
```

Comment:
Looks good, but you have several places named "pet". Do maybe do file search & replace all with "product"?